x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/pages/coverage_overview/coverage_overview_dashboard.tsx

```diff
@@ -14,26 +15,23 @@ import { CoverageOverviewMitreTechniquePanelPopover } from './technique_panel_po
 import { CoverageOverviewFiltersPanel } from './filters_panel';
 import { useCoverageOverviewDashboardContext } from './coverage_overview_dashboard_context';
 
+const CoverageOverviewSubtitle = (
+  <EuiText color="subdued" size="s">
+    <span>{i18n.CoverageOverviewDashboardInformation}</span> <CoverageOverviewLink />
+  </EuiText>
+);
+
 const CoverageOverviewDashboardComponent = () => {
   const {
     state: { data },
   } = useCoverageOverviewDashboardContext();
-  const subtitle = (
-    <EuiText color="subdued" size="s">
-      <span>{i18n.CoverageOverviewDashboardInformation}</span>{' '}
-      <EuiLink
-        external={true}
-        href={'https://www.elastic.co/'} // TODO: change to actual docs link before release
-        rel="noopener noreferrer"
-        target="_blank"
-      >
-        {i18n.CoverageOverviewDashboardInformationLink}
-      </EuiLink>
-    </EuiText>
-  );
+
   return (
     <>
-      <HeaderPage title={i18n.COVERAGE_OVERVIEW_DASHBOARD_TITLE} subtitle={subtitle} />
+      <HeaderPage
+        title={i18n.COVERAGE_OVERVIEW_DASHBOARD_TITLE}
+        subtitle={CoverageOverviewSubtitle}
+      />
```

Comment:
Thanks for refactoring and creating this `CoverageOverviewSubtitle` component. As suggested in https://github.com/elastic/kibana/pull/164613#discussion_r1305895896, let's memoize because it's static markup that can be rendered only once. We could extract the whole header into a `CoverageOverviewPageHeader` and wrap it with `React.memo`.